app/controllers/api/v1/backgrounds_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V1::BackgroundsController < ApplicationController
+  def show
+    location = params[:location]
+    city = location.split(',')[0]
```

Comment:
Made the keyword just the city for search. Could use gem in the future to turn state code into full state name to use in search as well.